schemas/eq_submission_schema.json

```diff
@@ -208,13 +233,13 @@
       "title": "Case ID",
       "default": "",
       "examples": [
-        "42d78898-7e7a-40b9-9500-757bb546e358"
+        "5515cd5f-fcff-44f0-ac02-3530477693ea"
       ],
       "pattern": "^[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$"
     },
     "data": {
       "$id": "#/properties/data",
-      "type": "array",
+      "type":  "object",
```

Comment:
Shouldn't this now have `data` and `lists` properties?

Reply:
We've incorporated lists to data section now, can you check please?